src/plugins/expressions/public/loader.test.ts

```diff
@@ -159,7 +159,7 @@ describe('ExpressionLoader', () => {
         throttle: 20,
       });
 
-      expectObservable(expressionLoader.data$).toBe('a 19ms c 2ms d', {
+      expectObservable(expressionLoader.data$).toBe('a 19ms c 19ms d', {
```

Comment:
This is a performance regression that breaks partial results specs. It seems to be a breaking change on the RxJS side. It will significantly slow down expressions invocation when `partial` and `throttle` flags are enabled simultaneously.

Reply:
@dokmic I'm not very familiar with the code. Would you mind helping me out here? Is https://github.com/elastic/kibana/issues/129462 a follow-up issue to cover this?

Reply:
@afharo, yes, that's right. I will take care of that when this PR is merged.